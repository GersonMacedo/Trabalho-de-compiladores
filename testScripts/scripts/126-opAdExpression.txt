! -s SYNTACTIC
! should accept expressions with opAd
program p ;
begin
    a := c + 39;
    a := 9 - a;
    a := false or true;
    a := 99 + a + 5 + 0;
    a := 1001 - 4 - c - 10;
    a := a + 49 - c + 0 - 3;
    a := false or t or true or j;
end .