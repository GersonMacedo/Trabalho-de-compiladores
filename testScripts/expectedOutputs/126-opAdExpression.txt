
Performing up to syntactic analysis at testScripts/scripts/126-opAdExpression.txt
No errors found
